Makefile

```diff
@@ -180,7 +181,7 @@ test_nodejs:: provider install_nodejs_sdk
 
 test_nodejs_upgrade:: PATH := $(WORKING_DIR)/bin:$(PATH)
 test_nodejs_upgrade:: provider install_nodejs_sdk
-	cd provider && go test -tags=nodejs -v -json -count=1 -cover -timeout 3h -parallel ${TESTPARALLELISM} . 2>&1 | tee /tmp/gotest.log | gotestfmt
+	cd provider && go test -tags=nodejs -v -json -count=1 -cover -timeout 3h -parallel 4 . 2>&1 | tee /tmp/gotest.log | gotestfmt
```

Comment:
Setting it to 4 still completed in 9 minutes. I kept getting terminated errors when set to 12 which I think were caused by resource issues.